Beeper/BarcelonaMautrixIPC/IPCHandlers/BLEventHandler.swift

```diff
@@ -1,5 +1,5 @@
 //
-//  BLEventHandler.swift
+//  self.ipcChannel.writePayload.swift
```

Comment:
For some reason, I think this may not have been intentional

Reply:
One day I'll learn how to use Xcode